source/basic.tex

```diff
@@ -3056,7 +3056,11 @@
 A byte is at least large enough to contain any member of the basic
 \indextext{character set!basic execution}%
 execution character set\iref{lex.charset}
-and the eight-bit code units of the Unicode UTF-8 encoding form
+and the eight-bit code units of the Unicode%
+  \footnote{Unicode\textregistered\ is a registered trademark of Unicode, Inc.
+  This information is given for the convenience of users of this document and
+  does not constitute an endorsement by ISO or IEC of this product.}
```

Comment:
I assumed from https://github.com/cplusplus/nbballot/issues/399 that this only needs to appear the first time, which already does with its resolving PR https://github.com/cplusplus/draft/pull/4175.

Reply:
I see that you moved this text from intro.tex to here, even though intro.tex appears first in the PDF. If their appearance in intro.tex doesn't count as the first time, shouldn't the other trademarks be moved accordingly? ECMA's to [tab:re.synopt] and POSIX's to, maybe, [namespace.posix]. Should I open an issue?

Reply:
We are moving the reference to UAX#29 from the front matter in [intro.refs] to the back of the standard in the bibliography.
The footnote explaining that "Unicode" is a trademark is now attached to the first appearance of "Unicode" in the standard, which is no longer the bibliographical reference, but the naming of Unicode in [basic].
In contrast, ECMA and POSIX references remain in [intro.refs] and thus the trademark footnote stays there.